<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchDenoiseDlg</class>
 <widget class="QDialog" name="BatchDenoiseDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Folder Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFolPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindFolder">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Batch size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbBatchSz">
       <property name="maximum">
        <number>16384</number>
       </property>
       <property name="singleStep">
        <number>32</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: Will find all WAVs, recursive (folders and subfolders), and REPLACE FILES. If you don't want it to do that, make a copy first. Treats all files as mono.&lt;/p&gt;&lt;p&gt;Note 2: All files have an output sampling rate of 48KHz (48000). If the input rate is different, they are automatically resampled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFiles">
     <property name="text">
      <string>Files: 0 / 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pgFiles">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
